libkbfs/backpressure_disk_limiter.go

```diff
@@ -22,19 +22,22 @@ import (
 // Let J be the (approximate) byte usage of the journal and F be the
 // free bytes on disk. Then we want to enforce
 //
-//   J <= min(J+F, L),
+//   J <= min(k(J+F), L),
 //
-// where L is the absolute byte usage limit. But in addition to that,
-// we want to set thresholds 0 <= m <= M <= 1 such that we apply
-// proportional backpressure (with a given maximum delay) when
+// where 0 < k <= 1 is some fraction, and L > 0 is the absolute byte
+// usage limit. But in addition to that, we want to set thresholds 0
+// <= m <= M <= 1 such that we apply proportional backpressure (with a
+// given maximum delay) when
 //
-//   m <= max(J/(J+F), J/L) <= M.
+//   m <= max(J/(k(J+F)), J/L) <= M.
 type backpressureDiskLimiter struct {
 	log logger.Logger
 	// backpressureMinThreshold is m in the above.
 	backpressureMinThreshold float64
 	// backpressureMaxThreshold is M in the above.
 	backpressureMaxThreshold float64
+	// maxJournalByteUsage is k in the above.
+	maxJournalByteUsage float64
```

Comment:
Can you throw a `Frac` or something similar in this field name (and corresponding parameter name) somewhere?  With this names it seems like it should be in bytes.